plugins/scaffolder-backend/src/lib/templater/cookiecutter.ts

```diff
@@ -0,0 +1,34 @@
+import { TemplaterBase, TemplaterRunOptions } from '.';
+
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as fs from 'fs-extra';
+
+class CookieCutter implements TemplaterBase {
+  public async run(options: TemplaterRunOptions): Promise<string> {
+    // first we need to make cookiecutter.json in the directory provided with the input values.
+    const cookieInfo = {
+      _copy_without_render: ['.github/workflows/*'],
+      ...options.values,
+    };
+
+    await fs.writeJSON(options.directory, cookieInfo);
+    return '';
+    // run cookie cutter with new json
```

Comment:
This ends up being pretty tricky with the current architecture, since we'd have to add cookiecutter to the docker image. What do you think @freben?

Reply:
I mean - for now I think it's fine. Let's keep it simple. Otherwise we could use something like Redis for some form of queue provider, and just push to redis and let some other renderers pick up the messages and write to some datastore. That just feels like overengineering right now to me though which is why I didnt think it would be too problematic right now.

Reply:
No I mean even keeping it simple is tricky xP. But I guess we just say that the cookiecutter command has to be part of the environment where this plugin is used.

Reply:
Yeah that way we can punt on it. But we'll have to decide and document recommendations on how people make use of external dependencies. Especially if we include a builtin packaging step.